modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -53,14 +56,17 @@ public ListTypeEntry addListTypeEntry(
 			long listTypeDefinitionId, String key, Map<Locale, String> nameMap)
 		throws PortalException {
 
+		User user = _userLocalService.getUser(userId);
+
+		_validateExternalReferenceCode(
+			externalReferenceCode, user.getCompanyId(), 0L);
+
```

Comment:
Could you try to delete this empty line here and run the SF? I believe this should not exist this blank line.